src/components/Modal.js

```diff
@@ -0,0 +1,140 @@
+import { useState, useEffect } from 'react';
+import './Modal.css';
+import { getImage } from '../helper/helps';
+import { useDispatch, useSelector } from 'react-redux'
+import { fetchPoken, initCompare, clearModal } from '../store/sliceModal';
```

Comment:
Import statements must have an order. Like this:
React imports should go first, followed by libraries and external tools (like react-redux in this case), followed by components and utils from your project and CSS files at the end.

There are tools that can do this for you. This is also part of formatting the code.